### PR TITLE
Sync Auditing to Develop

### PR DESCRIPTION
**Pull request by forrest-keller**
This branch needs to be synced with develop before merging.

### CODE REVIEW

File: src/routes/courses.ts [hunk 1]
```diff
@@ -1,8 +1,9 @@
-import { DayOfWeek, Department, Term } from "@prisma/client";
```
Comment (christianTatu) [inline src/routes/courses.ts:1]: @forrest-keller do we want to import term or no?

File: src/routes/courses.ts [hunk 2]
```diff
@@ -11,67 +12,48 @@ router.get(
...
+      next(err);
+    }
   }
-    
 );
 
 router.post("/", authorize(["ADMINISTRATOR"]), async (req, res, next) => {
   const schema = z.object({
     name: z.string(),
-    term: z.nativeEnum(Term),
```
Comment (christianTatu) [inline src/routes/courses.ts:74]: should this be changed?

Comment (christianTatu) [reply src/routes/courses.ts:74]: @forrest-keller

File: src/routes/courses.ts
Comment (christianTatu) [inline src/routes/courses.ts:1]: @forrest-keller @LoganMartinez looks like changes to making term it's own object, can yall double check this routes file to make sure it's all kosher? everything good besides that.

**Review by christianTatu — COMMENTED** [review]
Just need courses checked then all good

File: src/routes/courses.ts (hunk 1 above)
Comment (LoganMartinez) [reply src/routes/courses.ts:1]: no

File: src/routes/courses.ts (hunk 2 above)
Comment (LoganMartinez) [reply src/routes/courses.ts:74]: yeah it should be termId: z.string()

File: src/routes/courses.ts
Comment (LoganMartinez) [reply src/routes/courses.ts:1]: looks good after making those changes. If it compiles file then it should be good to go

**Review by christianTatu — APPROVED** [review]

**Pull request merged**